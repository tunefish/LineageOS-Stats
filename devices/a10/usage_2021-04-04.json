{
  "10.3": 1,
  "16.0": 68,
  "17.1": 834,
  "18.1": 360,
  "none": 348
}