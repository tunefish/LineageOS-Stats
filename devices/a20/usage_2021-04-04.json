{
  "17.1": 2431,
  "18.1": 180,
  "none": 28
}