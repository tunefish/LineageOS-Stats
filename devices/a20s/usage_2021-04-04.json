{
  "17.1": 2491,
  "none": 1
}