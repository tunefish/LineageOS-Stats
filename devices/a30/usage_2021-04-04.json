{
  "17.1": 35,
  "18.0": 735,
  "18.1": 391,
  "none": 4
}