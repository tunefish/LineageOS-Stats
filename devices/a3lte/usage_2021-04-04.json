{
  "15.1": 715,
  "16.0": 256,
  "17.1": 858
}