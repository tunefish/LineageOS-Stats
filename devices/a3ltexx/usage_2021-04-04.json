{
  "14.1": 1094
}