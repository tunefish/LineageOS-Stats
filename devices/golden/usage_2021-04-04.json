{
  "13.0": 1249,
  "14.1": 4320,
  "15.1": 5,
  "17.1": 8,
  "none": 1
}